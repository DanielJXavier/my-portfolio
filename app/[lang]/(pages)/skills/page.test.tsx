import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import { useParams, useSearchParams } from "next/navigation";

import en from "dictionaries/en.json";

import Page, { generateMetadata } from "./page";

jest.mock("next/navigation");

(useParams as jest.Mock).mockImplementation(() => ({ lang: "en" }));

(useSearchParams as jest.Mock).mockImplementation(() => ({
  get: jest.fn().mockImplementation((_) => null),
}));

describe("Skills page (Server-side)", () => {
  it("renders the page", () => {
    const { container } = render(<Page />);

    expect(container).toMatchSnapshot();
  });

  it("generate the page metadata", async () => {
    const metadata = await generateMetadata({
      params: {
        lang: "en",
      },
    });

    const {
      skills: { title },
    } = en;

    expect(metadata.title).toEqual(title);
  });
});
