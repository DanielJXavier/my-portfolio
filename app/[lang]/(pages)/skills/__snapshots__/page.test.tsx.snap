// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Skills page (Server-side) renders the page 1`] = `
<div>
  <h1
    class="text-xl md:text-2xl lg:text-3xl flex items-center gap-x-2"
    data-testid="title"
  >
    <svg
      class="fill-primary size-6 md:size-8"
      viewBox="0 -960 960 960"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M260-100v-161.23q-57-52-88.5-119.19-31.5-67.2-31.5-139.86 0-141.55 99.17-240.63Q338.33-860 480-860q116.54 0 208.42 69.66 91.89 69.65 119.35 180.73l48.96 194.12q3.5 13.64-4.92 24.57Q843.38-380 828.77-380H740v142.31q0 23.5-17.1 40.59-17.09 17.1-40.59 17.1H580v80h-45.39v-125.39h147.7q5.38 0 8.84-3.46t3.46-8.84v-187.7h112.47L764-598.85q-23.62-95.84-103.46-155.8-79.85-59.96-180.54-59.96-122.77 0-208.69 85.04t-85.92 207.49q0 63.55 25.93 120.29 25.94 56.73 73.14 100.6l20.93 19.88V-100H260Zm236.23-357.31Zm-40.69 78.69h47.69l2.23-39q14.69-2.38 27.9-9.98 13.21-7.6 22.72-18.4l36.23 14 22.61-38.38-27.31-21.7q6.16-16.43 6.16-33.87 0-17.43-6.16-33.74l27.31-21.69-22.61-38.39-36.23 14q-9.87-10.38-23.21-17.69-13.33-7.31-27.41-10.31l-2.23-39.38h-47.69l-2.23 39.38q-14.46 3-27.8 10.31-13.33 7.31-23.2 17.69l-36.23-14-22.23 38.39 27.3 21.69q-6.53 16.43-6.53 33.87 0 17.44 6.53 33.74l-27.3 21.7L366.08-432l36.23-14q9.51 10.8 22.71 18.4 13.21 7.6 28.29 9.98l2.23 39Zm23.94-72.46q-31.4 0-53.06-21.73-21.65-21.74-21.65-52.79 0-31.4 21.56-53.05 21.56-21.66 52.77-21.66 31.21 0 53.05 21.56Q554-557.19 554-525.98q0 31.21-21.73 53.06-21.74 21.84-52.79 21.84Z"
      />
    </svg>
    Skills
  </h1>
  <menu
    class="mt-1 md:mt-2 text-sm lg:mt-3 lg:text-base flex gap-x-1"
    data-testid="menu"
  >
    <li>
      <button
        class="text-secondary underline cursor-default hover:text-secondary hover:underline"
        disabled=""
      >
        all
      </button>
    </li>
    <li>
      |
    </li>
    <li>
      <button
        class=" hover:text-secondary hover:underline"
      >
        hard
      </button>
    </li>
    <li>
      |
    </li>
    <li>
      <button
        class=" hover:text-secondary hover:underline"
      >
        soft
      </button>
    </li>
  </menu>
  <article
    class="my-6 md:my-8 lg:my-10 flex gap-x-3 md:gap-x-5 lg:gap-x-8 gap-y-2 md:gap-y-3 lg:gap-y-4 flex-wrap items-center justify-center lg:justify-between"
  >
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      React
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Next.js
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Tailwind CSS
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      HTML5
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      CSS3
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Vue.js
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Nuxt.js
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Angular
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Responsive Design
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      SEO
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      SSR
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Sass
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Semantical HTML
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Webpack
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Redux
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      jQuery
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Web Accessibility
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Core Web Vitals
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Web Performance
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Web Security
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Web Sockets
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Styled Components
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      PWA
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Storybook
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Micro Frontends
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      TypeScript
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      JavaScript
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      MERN stack
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      MACH architecture
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Node.js
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Nest.js
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Express
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      LoopBack
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Meteor
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Kong
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Python
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      .NET
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Unit tests
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Jest
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Code Coverage
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      React Testing Library
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Cypress
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Chromatic
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Playwright
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      HTTP
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      GraphQL
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      REST
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Contentful
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      PHP
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      WordPress
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Magento
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Laravel
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Nginx
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Apache
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Linux Servers
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Windows Servers
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Stripe
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      CI/CD
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      GitHub Actions
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Bash
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Docker
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      YML
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Amazon Web Services (AWS)
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Google Cloud Platform (GCP)
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Amazon EC2 - Cloud Compute Capacity
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      AWS RDS (Relational Database Service)
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Amazon S3 - Cloud Object Storage
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      AWS Lambda
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Amazon CloudFront - Content Delivery Network
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Kubernets
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Terraform
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Vercel
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      SQL
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      MySQL
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      SQL Server
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      PostgreSQL
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Oracle
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      NoSQL
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      MongoDB
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Jira
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Confluence
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Slack Workflows
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Slack Apps
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      BPMN
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Git
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      A/B Testing
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Atomic Design
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Design Systems
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Semantic Versioning
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      UI/UX
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Google Ad Manager
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Agile
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Scrum
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Kanban
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Native Android
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Java
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Kotlin
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Native iOS
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      SwiftUI
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      React Native
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Firebase
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Functional Programming
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      OOP
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Regular Expressions
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      MarkDown
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Elasticsearh
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Delphi
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Leadership
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Communication
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Problem Solving
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Decision Making
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Empathy
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Conflict Resolution
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Adaptability
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Time Management
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Organization
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Strategic Thinking
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Mentoring
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Negotiation
    </span>
    <span
      class="text-3xl md:text-5xl lg:text-6xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Resilience
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Critical Thinking
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Creativity
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Active Listening
    </span>
    <span
      class="text-base md:text-xl lg:text-2xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Team Building
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Ethical Decision Making
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Customer Orientation
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Stakeholder Management
    </span>
    <span
      class="text-xl md:text-2px	lg:text-3xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Learning Agility
    </span>
    <span
      class="text-2xl md:text-4xl lg:text-5xl font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Curiosity
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Patience
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Public Speaking
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Networking
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Multitasking
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Attention to Detail
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Motivation
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Persuasion
    </span>
    <span
      class="text-sm md:text-base lg:text-lg font-light tracking-tighter [word-spacing:-0.35em]"
    >
      Emotional Intelligence
    </span>
  </article>
  <article
    class="py-5 md:py-6 lg:py-8 text-base md:text-lg lg:text-2xl font-light text-center lg:text-justify"
  >
    And always learning new things to expand the list...
  </article>
</div>
`;
